.github/workflows/edu-deploy-crawler.yml

```diff
@@ -21,11 +21,16 @@ on:
         required: true
       KUBECONFIG:
         required: true
+      PROXY_CREDENTIALS:
+        required: false
 
 jobs:
   run_ansible_playbook_edusharing:
     environment: ${{ inputs.CLUSTER }}
     runs-on: ubuntu-latest
+    env:
+      K8S_AUTH_PROXY: ${{ vars.PROXY_URL }}
```

Comment:
Where are the variables used?

Reply:
They are used by the kubernetes.core.k8s module in ansible.

Reply:
Could you please add the reference as a comment.